pom.xml

```diff
@@ -102,17 +92,15 @@
         </dependency>
 
         <dependency>
-            <groupId>org.mule.tests</groupId>
-            <artifactId>mule-tests-runner</artifactId>
-            <version>${muleTestsRunnerVersion}</version>
-            <scope>test</scope>
+            <groupId>com.sun.activation</groupId>
+            <artifactId>javax.activation</artifactId>
+            <version>1.2.0</version>
         </dependency>
 
         <dependency>
-            <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-file-extension-common</artifactId>
-            <version>${muleFileCommonsVersion}</version>
-            <type>test-jar</type>
+            <groupId>org.mule.tests</groupId>
+            <artifactId>mule-tests-runner</artifactId>
+            <version>${muleTestsRunnerVersion}</version>
             <scope>test</scope>
         </dependency>
 
```

Comment:
I think we have missed an exclusion in mule-tests-infrastructure.

`<exclusions>
                <exclusion>
                    <artifactId>bcprov-jdk15on</artifactId>
                    <groupId>org.bouncycastle</groupId>
                </exclusion>
            </exclusions>`